controllers/board-game-review-controllers.js

```diff
@@ -44,3 +44,16 @@ exports.getComments = (req, res, next) => {
         next(err);
     });
 };
+
+exports.newComment = (req, res, next) => {
+    const comment = req.body
+    const id = req.params.review_id
+
+    Promise.all([isCommentValid(comment), insertComment(comment, id)])
```

Comment:
I like the use of validating your comment body here - you may find that the database would help here if anything was invalid also